pandas/tests/arrays/categorical/test_algos.py

```diff
@@ -64,6 +64,7 @@ def test_isin_cats():
     [
         ("b", "c", ["a", "c"], "Categorical.categories are different"),
         ("c", "d", ["a", "b"], None),
+        ("a", "a", ["a", "b"], None),
```

Comment:
can you annotate this case (add a comment on line before with this issue number)